pkgs/development/python-modules/pleroma-bot/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, oauthlib
+, requests_oauthlib
+, requests
+, pyaml
+}:
+
+buildPythonPackage rec {
+  pname = "pleroma-bot";
+  version = "0.8.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "85056df457a49f2adbd36349d821c9401003d91a753de6456ceb805770e38c13";
+  };
+
+  doCheck = false;
```

Comment:
Can you add a comment with the reason for why checks have to be disabled?

Reply:
I was a bit confused as to what was the best source for the package. If I retrieved the archive from PyPI, no tests were included.
I've now changed it to ```fetchFromGitHub``` and added the pytest tests.